README.rst

```diff
@@ -1,7 +1,17 @@
+.. image:: https://readthedocs.org/projects/zopeappapidoc/badge/?version=latest
+        :target: http://zopeappapidoc.readthedocs.io/en/latest/?badge=latest
```

Comment:
`http` is what you get if you click the badge icon on RTD and copy-and-paste, which is what I did here. Presumably I then copy-and-pasted into CHANGES too.

I don't know if this means RTD prefers http (less overhead on the infrastructure?) or if they don't care. Anyway, I'll change it, thanks for the catch.

![untitled](https://user-images.githubusercontent.com/1256082/28120136-c9d96a0c-66dc-11e7-9cc5-a3d2ef164849.png)